v2/oauth/multiple-frontends-with-separate-backends.mdx

```diff
@@ -0,0 +1,286 @@
+---
+title: Multiple frontend domains with separate backends
+hide_title: true
+---
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import OAuthFrontendTabs from "/src/components/tabs/OAuthFrontendTabs"
+import OAuthBackendTabs from "/src/components/tabs/OAuthBackendTabs"
+import CoreInjector from "/src/components/coreInjector"
+import AppInfoForm from "/src/components/appInfoForm"
+import TabItem from '@theme/TabItem';
+
+# Multiple frontend domains with separate backends
+
+You can use the following guide if you have a single [**Authorization Service**](/docs/oauth/introduction#authorization-service) that is used by multiple applications.
+In turn, each app has separate **`frontend `** and **`backend`** instances that are served from different domains.
+Using the actual **OAuth 2.0** terminology, each application can be considered a [**Client**](/docs/oauth/introduction#client) while the **`backends`** are also [**Resource Servers**](/docs/oauth/introduction#resource-server).
+
+Note that if the *frontends* and *backends* are in different *sub domains*, you don't need to use *OAuth* and can instead use [session sharing across sub domains](/docs/session/common-customizations/sessions/share-sessions-across-sub-domains).
+
+<img src="/img/oauth/multiple-frontend-domains-with-separate-backends.png" alt="Multiple Frontend Domains with separate Backends"/>
+
+The authentication flow will work in the following way:
+
+1. User accesses the `frontend` app
+2. The application `frontend` calls a login endpoint on the `backend` application.
+3. The `backend` application generates an `authorization` URL to the [**Authorization Service**](/docs/oauth/introduction#authorization-service) and redirects the user to it.
+4. The [**Authorization Service**](/docs/oauth/introduction#authorization-service) backend redirects the user to the login UI
+5. User completes the login attempt 
+6. The [**Authorization Service**](/docs/oauth/introduction#authorization-service) backend redirects the user to a `callback URL` that includes the **Authorization Code**.
+7. The **Authentication Code** gets sent to the application `backend` when the user agent accesses the callback URL
+8. The `backend` exchanges the **Authentication Code** for an [**OAuth2 Access Token**](/docs/oauth/introduction#oauth2-access-token)
+9. The `backend` saves the received token in a server session and sends it back to the `frontend` as a cookie.
+10. The `frontend` uses the new cookie to access protected resources from the `backend`.
+
+:::info
+This guide assumes that you already have setup and configured **SuperTokens** in your [**Authorization Service**](/docs/oauth/introduction#authorization-service). 
+
+For more information on how to do that please check our [quickstart guides](/docs/guides).
+:::
+
+## 1. Enable the OAuth2 features from the Dashboard
```

Comment:
Also, you can use copy section feature in our docs so that this section which is same across all use cases docs, can be copied automatically. So any changes here will be reflected if you make change to the source of it.

To see its use search for `<COPY SECTION>` in our docs

Reply:
resolved